proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -37,9 +37,26 @@ import "temporal/server/api/enums/v1/common.proto";
 import "temporal/server/api/enums/v1/task.proto";
 import "temporal/server/api/history/v1/message.proto";
 import "temporal/server/api/namespace/v1/message.proto";
+import "temporal/server/api/replication/v1/message.proto";
 import "temporal/server/api/persistence/v1/executions.proto";
+import "temporal/server/api/persistence/v1/namespaces.proto";
 import "temporal/server/api/persistence/v1/workflow_mutable_state.proto";
-import "temporal/server/api/replication/v1/message.proto";
+
+message GetNamespacesRequest {
+    int32 page_size = 1;
+    bytes next_page_token = 2;
+}
+
+message GetNamespacesResponse {
+    repeated DescribeNamespaceResponse namespaces = 1;
+    bytes next_page_token = 2;
```

Comment:
APIs using pagination token should be called List*

Reply:
addressed and also here https://github.com/temporalio/temporal/pull/2159